central/reports/scheduler/v2/reportgenerator/csv_gen.go

```diff
@@ -50,7 +60,11 @@ func GenerateCSV(cveResponses []*ImageCVEQueryResponse, configName string, repor
 			r.GetDiscoveredAtImage(),
 			r.Link,
 		}
-		addOptionalColumnstoRow(reportFilters, &row, csvWriter, r)
+		if reportSnapshot.GetVulnReportFilters() != nil {
+			addOptionalColumnstoRow(reportFilters, &row, csvWriter, r)
+		} else {
+			addOtherColumns(&row, csvWriter, r)
+		}
```

Comment:
nit: I think it would be good to keep the checks consistent. Above you check using
``` 
if reportSnapshot.GetViewBasedVulnReportFilters() != nil {
      // do something
} else {
    // do other thing
}
```

It would be consistent to have this follow the same pattern too